survey/overview/time_random.rst

```diff
@@ -0,0 +1,34 @@
+============================
+Time and Randomize Questions
+============================
+
+Time Limit
+==========
+
+| On timed surveys, respondents need to complete the survey within a certain period of time. It
+  creates an impact on the collection of data, as the timing controls the pace at which the survey
+  is taken.
+| Set the *Time limit* under the tab *Options*.
```

Comment:
"It creates an impact on the collection of data" is weird 

I'd change it to something more about the fact that you want to limit the time they have to complete the test